CMakeLists.txt

```diff
@@ -571,3 +571,9 @@ endforeach ()
 target_sources(skeleton_test PRIVATE hwy/examples/skeleton.cc)
 
 endif()  # BUILD_TESTING
+
+# write HWYConfig file:
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file("${CMAKE_CURRENT_BINARY_DIR}/HWYConfigVersion.cmake" COMPATIBILITY SameMajorVersion)
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/HWYConfigVersion.cmake" DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/hwy")
+install(EXPORT hwy_targets FILE HWYConfig.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/hwy")
```

Comment:
The export should add a namespace (eg. `HWY::`). This lets downstream CMake recognize targets in link library properties and provide proper diagnostics at CMake generation time. 
(Plain literals like `hwy` could be namespecs to be passed to the linker. When the user fails to properly do `find_package(HWY)`, the lack of `hwy` can only be noticed at compile time or link time.)